runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargeType.java

```diff
@@ -26,6 +26,7 @@
 
 enum ItemChargeType
 {
+	BELLOW,
```

Comment:
Why is this named BELLOW and not BELLOWS?

Reply:
I was thinking the same thing when I was working on it, a fix will be pushed in a few minutes.